SIPS/sip-12.md

```diff
@@ -13,7 +13,7 @@ This SIP proposes a new endowment, `endowment:name-lookup`, that enables a way f
 
```

Comment:
This will be a lot of breaking changes to the existing API in Flask, I'm sure you are aware, but flagging anyway

Reply:
Yup yup